app/src/main/res/layout/fragment_counter.xml

```diff
@@ -4,47 +4,57 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".view.FragmentCounter">
-
+    tools:context=".view.CounterFragment">
 
     <TextView
-        android:id="@+id/viewmodel_counter"
+        android:id="@+id/textviewFragmentCounter"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginBottom="@dimen/dp_16"
-        app:layout_constraintBottom_toTopOf="@+id/fragment_counter"
+        app:layout_constraintBottom_toTopOf="@+id/textviewViewmodelCounter"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        tools:text="TextView" />
+        tools:text="@string/vm_text" />
 
     <TextView
-        android:id="@+id/fragment_counter"
+        android:id="@+id/textviewViewmodelCounter"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:text="TextView" />
+        tools:text="@string/fragment_text" />
```

Comment:
Task dokumanina baktigimda tasarimda tek textview uzerinden sayac degeri tutulmus gozukuyor. Burada iki farkli textview uzerinden ilerlemenin bir sebebi var mi?
https://github.com/mobillium-android/interns-2024/tree/main/task-3#ui-tasar%C4%B1m

Reply:
Abi aslında onu iki veriyi de altlı üstlü inceledeğimizde viewmodelin direkt korunduğunu fragmenttaki değişkenin de sıfırlandığını göstermek için öyle yapmıştım yani kafamda öyle canlanmıştı, düzeltiyorum hemen